app/src/main/java/com/x8bit/bitwarden/ui/platform/components/navigation/BitwardenNavigationBarItem.kt

```diff
@@ -0,0 +1,78 @@
+package com.x8bit.bitwarden.ui.platform.components.navigation
+
+import androidx.annotation.DrawableRes
+import androidx.annotation.StringRes
+import androidx.compose.foundation.layout.RowScope
+import androidx.compose.material3.BadgedBox
+import androidx.compose.material3.Icon
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.NavigationBarItem
+import androidx.compose.material3.NavigationBarItemDefaults
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.style.TextOverflow
+import com.x8bit.bitwarden.ui.platform.components.badge.NotificationBadge
+import com.x8bit.bitwarden.ui.platform.components.util.rememberVectorPainter
+
+/**
+ * A custom Bitwarden-themed bottom app bar.
+ *
+ * @param labelRes The custom label for the navigation item.
+ * @param contentDescriptionRes The custom content description for the navigation item.
+ * @param selectedIconRes The icon to be displayed when the navigation item is selected.
+ * @param unselectedIconRes The icon to be displayed when the navigation item is not selected.
+ * @param isSelected Indicates that the navigation item is selected.
+ * @param onClick The lambda to be invoked when the navigation item is clicked.
+ * @param modifier A [Modifier] that you can use to apply custom modifications to the composable.
+ * @param notificationCount The notification count for the navigation item.
+ */
+@Composable
+fun RowScope.BitwardenNavigationBarItem(
```

Comment:
Do you picture ever needing this anywhere other than the main screen?

Reply:
I want it in a reusable space for the authenticator eventually.

Also makes some of my coloring easier.

Reply:
OK, sounds good!